SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+# Next steps: Set up the SpatialOS GDK Starter template
+
+Before setting up the SpatialOS GDK Starter template, you need to have followed:
+
+* [Getting started: 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
+* [Getting started: 2 - Get and build the SpatialOS Unreal Engine Fork]({{urlRoot}}/content/get-started/tutorial).
+
+If you haven't already, follow our [Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/dependencies) to try out the SpatialOS for Unreal GDK development experience by adding a new feature to a project and testing it across two servers.
+
+If you are ready to start developing your own game with the GDK, follow the steps below. 
+
+### Terms used on this page
+
+* `<Game>` - The directory that contains your project’s .uproject file and Source folder.
+* `<ProjectRoot>` - The directory that contains your `<Game>` directory.
+* `<YourProject>` - The name of your project and .uproject file (for example, `\<Game>\YourProject.uproject`).
+
+### Create a new project using the starter template
+
+After [building the Unreal Engine fork]({{urlRoot}}/content/get-started/build-unreal-fork), in **File Explorer**, navigate to `UnrealEngine\Engine\Binaries\Win64`and double-click UE4Editor.exe to open the Unreal Editor. 
+
+In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, click the New Project tab and select the C++ tab, then select the **SpatialOS GDK Starter** template. Choose a suitable directory and name for your project, then click **Create Project**. 
+
+>Note: When you create a project, the Unreal Engine automatically creates a directory named after the project name you entered. This page uses `YourProject` as an example project name.
+
+![The Unreal Engine Project Browser]({{assetRoot}}assets/set-up-template/template-project-browser.png)
```

Comment:
In this screenshot, the Project name is 'YourProject' and the folder is 'C:\Unreal Projects\SpatialOS\YourProject'. 
As mentioned in the above note, when the project is generated, the engine creates a directory named after the project name you entered. In the example screenshot, the generated project will be located in 'C:\Unreal Projects\SpatialOS\YourProject\YourProject\'. Can we redo the screenshot with the 'Folder' path not containing the project name 'YourProject'?

Reply:
Edited the screenshot: 
![template-project-browser](https://user-images.githubusercontent.com/24454135/52710756-0a842180-2f88-11e9-9721-161d11982c54.png)